src/utils/noenumarable.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Set a given property of the target to be no-enumerable
+ * Will be no listed on Object.keys and will be exclude by spread operator
```

Comment:
Consider rewording: `Will (not be) listed on Object.keys and will be (excluded) by spread operator`

Reply:
Fixed: Will (not be) listed on Object.keys and will be excluded ...